rsql/rcache.go

```diff
@@ -22,6 +22,18 @@ type rcache struct {
 	limit  int
 }
 
+// enabled is the source of truth whether the rcache is enabled or not.
+var enabled bool
```

Comment:
This variable is called rsql.enabled. We're not disabling "rsql" - we're disabling just the cache part. Suggest `var disableCache bool`.

Reply:
Ah good point.